week-03/main.js

```diff
@@ -4,6 +4,8 @@
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
 
+list.shift(1);
```

Comment:
The shift() method removes the first element from an array and returns that element. It does not take any arguments.